tools/code_format/envoy_build_fixer.py

```diff
@@ -71,20 +71,20 @@ def run_buildozer(cmds, contents):
         return r.stdout.decode('utf-8')
 
 
-# Add an Apache 2 license and envoy_package() import and rule as needed.
```

Comment:
same comment here, not sure envoy_package() should've been removed from here?

Reply:
Same as before. This tool lives in Envoy Core, which means `envoy_package()` will be automatically added when running the tool in Envoy Code. I updated the comment.